src/libraries/Microsoft.PowerFx.Interpreter/CustomFunction/CustomTexlFunction.cs

```diff
@@ -22,7 +23,7 @@ namespace Microsoft.PowerFx
     /// <summary>
     /// Internal adapter for adding custom functions. 
     /// </summary>
-    internal class CustomTexlFunction : TexlFunction
+    internal class CustomTexlFunction : TexlFunction, IAsyncTexlFunction999
```

Comment:
>IAsyncTexlFunction999 [](http://example.com/codeflow?start=54&length=21)

Adding this here lets EvalVisitor avoid having to specifically check for CustomTexlFunction